doc/contrib/release.rst

```diff
@@ -112,11 +112,12 @@ Release
    pip install arbor
    python -c 'import arbor; print(arbor.__config__)'
 
-7. Update spack package / Ebrains Lab
+7. Update spack package / Ebrains Lab / Opensourcebrain
 
-   -  first, update ``spack/package.py``. The checksum of the targz is the sha256sum.
-   -  Then, use the file to `make PR here <https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/arbor/package.py>`_
-   -  Make an MR `here <https://gitlab.ebrains.eu/technical-coordination/project-internal/devops/platform/ebrains-spack-builds/>`_
+   - first, update ``spack/package.py``. The checksum of the targz is the sha256sum.
+   - Then, use the file to `make PR here <https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/arbor/package.py>`_
+   - Make an MR `here <https://gitlab.ebrains.eu/technical-coordination/project-internal/devops/platform/ebrains-spack-builds/>`_
+   - Make a PR to the OSB `dockerfile <https://github.com/OpenSourceBrain/OSBv2/blob/master/applications/jupyterlab/Dockerfile>`_
```

Comment:
Not yet :) I clarified that the main thing to make sure is, is that the Notebook we have in their showcase still works. My idea is that it's better to starting pinning `pip` to an Arbor version, so that would be the first PR.